vendor/engines/secure_rooms/app/controllers/secure_rooms_api/events_controller.rb

```diff
@@ -3,10 +3,40 @@ module SecureRoomsApi
   class EventsController < SecureRoomsApi::ApiController
 
     def create
-      Rails.logger.info "SecureRoomsApi Event: #{params}"
+      # TODO: Discuss how to handle errors
```

Comment:
I think this is probably fine for now. None of these fields are validated, so there _shouldn't_ be any problems. I think we can remove the TODO.